pages/pipelines/command_step.md.erb

```diff
@@ -140,12 +179,22 @@ steps:
       - "coverage/**/*"
     parallelism: 1
     timeout_in_minutes: 60
+    retry:
+      automatic:
+        - exit_status: 1
```

Comment:
It might be useful to document the `-1` use case - where there was actually no exit code reported, likely because of an infrastructure issue.

Reply:
Ah yeah, good call! @harrietgrace where do you reckon we put that?

Reply:
I'm adding in an 'optional attributes' section for `automatic`, I'll put a note in there!